pages/en/lb4/Schemas.md

```diff
@@ -9,9 +9,127 @@ summary:
 ---
 {% include content/tbd.html %}
 
-> Stub page per [#554](https://github.com/strongloop/loopback-next/issues/554).
+<!-- TOC -->
+
+- [Overview](#overview)
+- [Using Legacy Juggler](#using-legacy-juggler)
+  - [Model Decorator](#model-decorator)
+  - [Property Decorators](#property-decorators)
+  - [Defining Metadata By Hand?](#defining-metadata-by-hand)
+- [Other ORMs](#other-orms)
+
+<!-- /TOC -->
+
+
+## Overview
+A `schema` represents the definition of a model in LoopBack, with respect to
+the [datasource juggler](https://github.com/strongloop/loopback-datasource-juggler).
+Currently, we provide the `@loopback/repository` module, which provides special
+decorators for adding metadata to your TypeScript/JavaScript
+classes in order to use them with the legacy implementation of the Juggler.
+
+## Using Legacy Juggler
+Defining a schema for use with the legacy juggler involves decorating your
+classes with the `@model` and `@property` decorators.
+
+```ts
+import { model, property } from '@loopback/repository';
+
+@model()
+export class Product extends Entity {
+  @property({
+    type: 'number',
+    id: true,
+    description: 'The unique identifier for a product',
+  })
+  id: number;
+
+  @property({type: 'string'})
+  name: string;
+
+  @property({type: 'string'})
+  slug: string;
+
+  constructor(data?: Partial<Product>) {
+    super(data);
+  }
+}
+```
+
+### Model Decorator
+The model decorator can be used without any additional parameters, or can take
+in a ModelDefinitionSyntax object which follows the general format provided in
+LoopBack 3:
+```ts
+{
+@model({
+  name: "Foo",
+  properties: {
+    // define properties here.
+  },
+  settings: {
+    // etc...
+  }
+})
+class Foo {
+  // etc...
+}
+}
+```
+
+### Property Decorators
+The property decorator takes in the same arguments used in LoopBack 3 for
+individual property entries:
+```ts
+@model()
+class Foo {
+  @property({
+    name: "title",
+    description: "The title of the Foo",
+    type: "string",
+  })
+  public title: string;
+}
+```
+
+The complete list of attributes in property definition can be found in here:
+https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#properties
```

Comment:
Why are we linking to LB3 doc?  If this information is applicable to LB4, then we should provide it in the `lb4` folder, with any changes specific to LB4.

Reply:
This is for the legacy Juggler; as far as I know, we're not documenting the old legacy behaviour in LB 4
@raymondfeng @bajtos your thoughts?

Reply:
I should add that we're not yet set on what we intend for the new "juggler"; if we're supporting the same DSL as previously, we'll need to migrate documentation to that effect into the lb4 docs (and then this section would need re-writing to match that).

Regardless of what we go with, I'll make it a bit clearer that this is a link to the previous version's documentation.

Reply:
AFAIK, our current plan is to let `@loopback/repository@4.x` to depend on `loopback-datasource-juggler@3.x`, i.e. there is no v4 of juggler planned. It makes more sense to me to keep the content in a single place in `doc/en/lb3`.

What are the downsides of such solution?